### PR TITLE
🫥 [Linter fix] Remove absolute-imports since it is superseded by ruff TID252

### PR DESCRIPTION
**Pull request by trajepl**
## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Lint and apply fixes to your code by running `lintrunner -a`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.
- [ ] Is this PR including examples changes? If yes, please remember to update [example documentation](https://github.com/microsoft/Olive/blob/main/docs/source/examples.md) in a follow-up PR.

### CODE REVIEW

File: examples/test/azureml/test_bert_ptq_cpu_aml.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 import pytest
 
-from ..utils import check_output, get_example_dir, patch_config
+from examples.test.utils import check_output, get_example_dir, patch_config
```
Comment (jambayk) [inline examples/test/azureml/test_bert_ptq_cpu_aml.py:9]: does this import work if we want to run an example locally from a different directory that is not the repo root? like from `examples/test` itself?

Comment (jambayk) [reply examples/test/azureml/test_bert_ptq_cpu_aml.py:9]: If that works, can you also remove the TID252 rule from here https://github.com/microsoft/Olive/blob/798508b4d31d76362671d0c56b9bbce6b764a024/pyproject.toml#L193?

Comment (trajepl) [reply examples/test/azureml/test_bert_ptq_cpu_aml.py]: It did not work.

Just remove the checker in pre-commit config.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -18,11 +18,6 @@ repos:
     hooks:
       - id: isort
         name: Format imports
-  - repo: https://github.com/MarcoGorelli/absolufy-imports
-    rev: v0.3.1
-    hooks:
-      - id: absolufy-imports
-        exclude: examples/directml/llm/chat_app/
```
Comment (jambayk) [inline .pre-commit-config.yaml:25]: maybe keep this hook but exclude all of examples directory?

Comment (trajepl) [reply .pre-commit-config.yaml:25]: https://github.com/microsoft/Olive/pull/1272/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R193

Xiaoyu already updated this.

Comment (jambayk) [reply .pre-commit-config.yaml:25]: I meant to not delete this here. But just update the exclude part here to 'examples'. This way the pre commit hook can still fix the imports in other parts. 

Xiaoyu's change is for the lintrunner which is independent of the precommit hooks

**Review by jambayk — APPROVED** [review]

**Pull request merged**